.github/workflows/ci.yml

```
@@ -12,7 +12,13 @@ jobs:
         uses: actions/setup-java@v2
         with:
           java-version: '17'
-          distribution: 'adopt'
-          cache: maven
+          distribution: 'temurin'
+      - name: Cache Maven packages
+        uses: actions/cache@v2
```

actionlint (1.7.4)

the runner of "actions/cache@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)